bookwyrm/templates/book/book.html

```diff
@@ -48,10 +48,9 @@ <h2 class="subtitle">
 
     <div class="columns">
         <div class="column is-one-fifth">
-            <div class="is-clipped">
-                {% include 'snippets/book_cover.html' with book=book size=large %}
-                {% include 'snippets/rate_action.html' with user=request.user book=book %}
-            </div>
+            {% include 'snippets/book_cover.html' with book=book cover_class='has-height is-h-medium-mobile' %}
```

Comment:
What do you think about inserting the `book_cover` button on the cover itself, when the cover is `large` or `medium` only?

Reply:
do you mean the one to add the cover? I've been thinking for a while that it might be nice to have an "enlarge cover" button on the cover, which could be the add cover button if the cover is absent

Reply:
ooh yes, the dual mode button is a good idea

Reply:
I’ll give it a shot. :)

Reply:
Actually not… This PR is already quite big, so I will create a new Issue for that.

Reply:
Done: https://github.com/bookwyrm-social/bookwyrm/issues/1009